src/scenic/simulators/robosuite/simulator.py

```diff
@@ -0,0 +1,566 @@
+# src/scenic/simulators/robosuite/simulator.py
+
+"""RoboSuite Simulator interface for Scenic - Custom Environment Only."""
+
+from typing import Dict, List, Any, Optional, Union
+import numpy as np
+import mujoco
+
+try:
+    import robosuite as suite
```

Comment:
This import causes robosuite to print various warnings for me even if we won't actually use robosuite (e.g. if you run `scenic` without the `-S` option). Unless it would cause other problems, it might be better to not import robosuite at all unless it's going to be used. That's what we do for MetaDrive and CARLA: we set up their `model.scenic` files to work without needing to import the simulator packages, but their `simulator.py` files raise an exception if those packages are missing.